receiver/oracledbreceiver/scraper.go

```diff
@@ -430,6 +449,104 @@ func (s *oracleScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 	return out, nil
 }
 
+func (s *oracleScraper) scrapeLogs(ctx context.Context) (plog.Logs, error) {
+	const dbPrefix = "oracledb."
+	const queryPrefix = "query."
+	const duration = "DURATION_SEC"
+	const event = "EVENT"
+	const hostName = "MACHINE"
+	const module = "MODULE"
+	const osUser = "OSUSER"
+	const objectName = "OBJECT_NAME"
+	const objectType = "OBJECT_TYPE"
+	const process = "PROCESS"
+	const port = "PORT"
+	const program = "PROGRAM"
+	const planHashValue = "PLAN_HASH_VALUE"
+	const sqlID = "SQL_ID"
+	const schemaName = "SCHEMANAME"
+	const sqlChildNumber = "SQL_CHILD_NUMBER"
+	const sid = "SID"
+	const serialNumber = "SERIAL"
+	const status = "STATUS"
+	const state = "STATE"
+	const sqlText = "SQL_FULLTEXT"
+	const username = "USERNAME"
+	const waitclass = "WAIT_CLASS"
+
+	var scrapeErrors []error
+
+	dbClients := s.samplesQueryClient
+	if dbClients == nil {
+		s.logger.Info("Error creating client: ")
+		panic("exiting")
+	}
```

Comment:
```suggestion
	dbClients := s.samplesQueryClient
```

There is no need to check whether the client is usable, the check should happen when initializing the client.

And, do not use `panic` to handle the error, as `panic` might crash the entire application. Instead, return the error.